content/en/docs/Kyverno CLI/_index.md

```diff
@@ -216,6 +216,15 @@ policies:
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
 
+On applying the above policy to the mentioned resources, the following output will be generated:
+```sh
```

Comment:
Very minor thing, but in order to pass Markdown linting tests, fenced code blocks should be surrounded by a single new line.